pom.xml

```diff
@@ -42,42 +38,13 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
-		<dependency>
-			<groupId>com.fasterxml.jackson.core</groupId>
-			<artifactId>jackson-databind</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>com.fasterxml.jackson.core</groupId>
-			<artifactId>jackson-core</artifactId>
-		</dependency>
```

Comment:
Hey @iuliancorcoja why dont we need these ones anymore? i thought we used Jackson pretty heavily in our services?